packages/react-native-editor/src/setup.js

```diff
@@ -70,6 +70,8 @@ const setupInitHooks = () => {
 		) {
 			unregisterBlockType( 'core/block' );
 		}
+
+		doAction( 'native.post-register-core-blocks', props );
```

Comment:
I've double-checked that the props passed here are the ones expected as from the older approach in https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/src/index.js#L57-L66 👍 

I was wondering if we should remove the `native.render` action or keep it if it's needed in the future, what do you think? I'm fine with keeping it by the way.

Reply:
> I've double-checked that the props passed here are the ones expected as from the older approach in https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/src/index.js#L57-L66 👍

Thanks for checking it 🙇 ! I saw that we only use the `capabilities` prop when registering the Jetpack blocks, which is already being passed.

Reply:
> I was wondering if we should remove the `native.render` action or keep it if it's needed in the future, what do you think? I'm fine with keeping it by the way.

Good question. AFAIK it's not being used anywhere, so I understand we could remove it. If we need it in the future we can always revert it back.

Reply:
Sounds good to me! Are we planning to do that in a follow-up PR? This PR is ready for my approval so let me know!

Reply:
Ah, sorry. I'm planning to apply the suggestion in this PR. In fact, I noticed that I missed adding a unit test for the new hook. I'll let you know when I pushed the changes, thanks!

Reply:
Cool! I'll keep an eye out for those, thanks!

Reply:
@geriux I've applied you suggestion https://github.com/WordPress/gutenberg/pull/52791/commits/2edd4a86d7fdd35263fde07785fc997a9765cb17.

I also updated the unit tests in https://github.com/WordPress/gutenberg/pull/52791/commits/7fb8ba6c81284a29c01fc96a5a603559e563d52a. It took way longer than I expected because the test logic required some tricky changes to check the call orders. Let me know if you have any concerns/questions regarding that, thanks 🙇 !

Reply:
Thanks! I'll check it out